src/components/AlertBanner/AlertBanner.tsx

```diff
@@ -122,7 +122,7 @@ const AlertBanner: VibeComponent<AlertBannerProps> & {
               size={Button.sizes.SMALL}
               kind={Button.kinds.TERTIARY}
               color={isDarkBackground ? Button.colors.ON_INVERTED_BACKGROUND : Button.colors.ON_PRIMARY_COLOR}
-              ariaLabel="close-toast"
+              closeButtonAriaLabel="Close"
```

Comment:
You got it a bit wrong. Here it should be `ariaLabel={closeButtonAriaLabel}` and `closeButtonAriaLabel` should be a prop of `AlertBanner` with default value = `Close`. This way users will be able to pass other ariaLabel into the component e.g. when localization is needed and user is using the system in, let's say, German